docs/content/en/docs/packages/cluster-autoscaler/addclauto.md

```diff
@@ -101,24 +80,31 @@ To read more about the tradeoffs of these different approaches, see [Autoscaling
    eksa-packages-mgmt-v-vmc   cluster-autoscaler            cluster-autoscaler   18h   installed   9.21.0-1.21-147e2a701f6ab625452fe311d5c94a167270f365         9.21.0-1.21-147e2a701f6ab625452fe311d5c94a167270f365 (latest)
    ```
 
-   To verify that autoscaling works, apply this deployment:
+   To verify that autoscaling works, apply the deployment below. You must continue scaling pods until the deployment has pods in a pending state.
+   This is when cluster autoscaler will begin to autoscale your machine deployment.
+   This process may take a few minutes.
    ```bash
    kubectl apply -f https://raw.githubusercontent.com/aws/eks-anywhere/d8575bbd2a85a6c6bbcb1a54868cf7790df56a63/test/framework/testdata/hpa_busybox.yaml
    kubectl scale deployment hpa-busybox-test --replicas 100
    ```
-   You must continue scaling pods until the deployment has pods in a pending state.
-   This is when cluster autoscaler will begin to autoscale your machine deployment.
-   This process may take a few minutes.
+
 
 ## Update
 To update package configuration, update cluster-autoscaler.yaml file, and run the following command:
 ```bash
 eksctl anywhere apply package -f cluster-autoscaler.yaml
 ```
 
+## Update Worker Node Group Autoscaling Configuration
+It is possible to change the autoscaling configuration of a worker node group by updating the WorkerNodeGroup.autoscalingConfiguration in your cluster specification and running a cluster upgrade.
+
 ## Upgrade
 
-Cluster Autoscaler will automatically be upgraded when a new bundle is activated.
+The Cluster Autoscaler can be upgraded by PackageController's `activeBundle` field to a newer version.
```

Comment:
We should eventually have separate docs that explain the bundle, etc, but for now I'm putting a little explainer here.